pages/latest/[urlTitle].tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { GetStaticPathsResult, GetStaticPropsContext } from 'next';
+import { ParsedUrlQuery } from 'node:querystring';
+
+// Graphql
+import { NAVIGATION_MENU_QUERY, FULL_PAGE_QUERY } from '@graphql/queries';
+
+// Interfaces
+import { DotcmsDocumentation, NavigationProp } from '@models/dotcmsDocumentation.interface';
+
+// Utils
+import { client } from '@utils/graphql-client';
+
+const urlTitle = ({ data }: { data: DotcmsDocumentation[] }): JSX.Element => {
+    const documentation = data[0];
+    return <h1>{documentation.title}</h1>;
+};
+
+export async function getStaticPaths(): Promise<GetStaticPathsResult> {
+    const data = await client.request(NAVIGATION_MENU_QUERY);
+    const paths = buildParams(data.DotcmsDocumentationCollection[0], []);
+    return {
+        paths,
+        fallback: false
+    };
+}
+
+const buildParams = (data, paths): UrlTitleParams[] => {
+    if (!data.dotcmsdocumentationchildren?.length) {
+        return paths;
+    }
+    data.dotcmsdocumentationchildren.map((item: DotcmsDocumentation) => {
```

Comment:
`.map` returns an new array, what you need to use here is a `forEach` because you are creating the array yourself.